libkbfs/mdserver_remote.go

```diff
@@ -530,6 +530,10 @@ func (md *MDServerRemote) GetForHandle(ctx context.Context,
 		BranchID:     NullBranchID.String(),
 		Unmerged:     mStatus == Unmerged,
 	}
+	if lockBeforeGet != nil {
+		copied := *lockBeforeGet
```

Comment:
Why is the copy needed?  Isn't this just a `long`?  Once you make the RPC, the long gets serialized, and there's nothing hanging around that can get modified asynchronously.

Reply:
Oh good point. I'll remove the copy. Thanks!